File: parse_status.py
import json, os
try:
    docid = os.environ.get("DOCID")
    with open("doclist.json","r",encoding="utf-8") as f:
        lst = json.load(f)
    if isinstance(lst, list):
        for d in lst:
            if isinstance(d, dict) and d.get("id") == docid:
                print(d.get("status",""), end="")
                break
except Exception:
    pass

File: parse_status.js
const fs = require(fs); const id = process.env.DOCID;
try {
  const list = JSON.parse(fs.readFileSync(doclist.json,utf8));
  const it = Array.isArray(list) ? list.find(d => d && d.id === id) : null;
  if (it && it.status) process.stdout.write(it.status);
} catch(e) {}
